test/type-tests/bind-all.usage-types.ts

```diff
@@ -23,13 +23,13 @@ import { bindAll } from '../../src';
   bindAll(button, [
     {
       type: 'click',
-      listener(event: MouseEvent) {
+      listener(event) {
         const value: number = event.button;
       },
     },
     {
       type: 'keydown',
-      listener(event: KeyboardEvent) {
+      listener(event) {
```

Comment:
I can see this is now getting the correct type on `typescript5.0.0-beta`

Reply:
Autocomplete is working on `typescript5.0.0-beta`

Reply:
Auto completed not working in `typescript@4.9.5` (expected)

Reply:
Correct type inferred on `@typescript@4.9.5`

Reply:
> I can see this is now getting the correct type on typescript5.0.0-beta

The correct type should also be reported on 4.x

Reply:
So as expected: types are fixed, but autocomplete now only works in TS 5+